### PR TITLE
olly-wilson/Added OllyWilson component

### PR DESCRIPTION
**Pull request by Olly111**
### What did you do & How did you do it?
Copy Paste MichaelNguyen component and change appropriate values

### Why did you do it?
To satisfy the customer's specification (Mike)

### CODE REVIEW

**Review by AdamSchlicht — APPROVED** [review]
LGTM

**Review by Olly111 — COMMENTED** [review]
Cry havoc and let slip the dogs of merge hell 🤣

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
-import { MichaelNguyen } from './components/MichaelNguyen/MichaelNguyen';
+import logo from "./logo.svg";
```
Comment (Booligoosh) [inline src/App.js]: Double quotes = good 👍

**Review by Booligoosh — APPROVED** [review]
Looks good to me

File: src/App.js (hunk 1 above)
Comment (Olly111) [reply src/App.js]: Prettier got my back 🤣

**Pull request merged**